### ISSUE TITLE
Use sufficient color contrast ratios.

### ISSUE THREAD

**Issue by shavenking** [shavenking/money-mom-ios#60]
Use sufficient color contrast ratios. Insufficient contrast in your app makes content hard to read for everyone. Icons and text might blend with the background, for example. An online color contrast calculator can help you accurately analyze the color contrast in your app, to ensure that it meets optimal standards. Strive for a minimum contrast ratio of 4.5:1, although 7:1 is preferred because it meets more stringent accessibility standards.
